src/main/webapp/css/_bootstrap.scss

```diff
@@ -0,0 +1,10657 @@
+/*!
```

Comment:
I see this was deleted in another PR. Please update.

Reply:
fixed